jest/setup.js

```diff
@@ -8,6 +8,9 @@
  */
 'use strict';
 
+var immutable = require('immutable');
+var {Map} = immutable;
```

Comment:
I agree and have updated! It makes more sense to use `immutable.Map` (btw it looks like it's imported with a lowercase `i` in other places so I didn't change it).